app/keyboard.go

```diff
@@ -3,6 +3,19 @@ package app
 import "C"
 import "github.com/fyne-io/mobile/event/key"
 
+type Keyboard int32
```

Comment:
As this is a hint or type of keyboard rather than an actual reference to a keyboard it probably needs a name that is less concrete too. Like KeyboardType ...Hint or ...Mode?